carbon-design-system/carbon-addons-iot-react
# 3255 merge master back into next

Closes #3255

**Summary**
Merged master back into next

**Change List (commits, features, bugs, etc)**
- Solved merge conflicts for the table related to the refactoring of table story helper functions
- Solved merge conflicts for the versioning in the different `package.json` files, the `lerna.json` and the different `CHANGELOG.md` files.  
- Updated visual regression image for the table
- Updated snapshots

**Acceptance Test (how to verify the PR)**
- Verify the 4 fixes added to master after the original branch
- Verify the table stories
- Make sure the CHANGELOG files are correct

**Regression Test (how to make sure this PR doesn't break old functionality)**
- Verify the table stories



**Things to look for during review**
